docs/iceberg-rest-service-blog.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Gravitino Iceberg REST catalog service"
+slug: /iceberg-rest-service-blog
+keywords:
+  - Iceberg REST catalog
+license: "Copyright 2023 Datastrato Pvt Ltd.
+This software is licensed under the Apache License version 2."
+---
+
+## Background
+[Apache Iceberg](https://iceberg.apache.org/) is a popular high-performance format for huge analytic tables. Iceberg catalogs is responsible for creating, dropping, alerting Iceberg tables and namespaces. The catalog stores a reference to the current table metadata, and provides the grant of atomic update to the current table metadata. Iceberg catalog is completely pluggable and provides several implements:
```

Comment:
In Gravitino Iceberg catalog is responsible....
and alerting
"the grant of atomic update to the current table metadata." is not easily understood and needs to be rewritten
The Iceberg catalog...